SabreTools.DatTools/Rebuilder.cs

```diff
@@ -236,7 +246,12 @@ public class Rebuilder
                     foreach (string file in Directory.EnumerateFiles(input, "*", SearchOption.AllDirectories))
                     {
                         logger.User($"Checking file: {file}");
-                        bool rebuilt = RebuildGenericHelper(datFile, file, outDir, quickScan, date, inverse, outputFormat, asFiles);
+                        string tempDir = Path.GetDirectoryName(file.Replace(input,string.Empty));
+                        if (!String.IsNullOrEmpty(tempDir))
+                            tempDir += "/";
```

Comment:
Why are you adding this directory separator? And why not use the built-in directory separator instead?